requirements.txt

```diff
@@ -244,11 +243,12 @@ tldextract==3.1.0
     # via presidio-analyzer
 tokenizers==0.10.3
     # via transformers
-torch==1.8.1; sys_platform != '*win*'
+torch==1.8.1
```

Comment:
This is manual step I always had to do it :(, please add `; sys_platform != '*win*'` otherwise windows build will fail.

Reply:
done